extension/llm/runner/io_manager/io_manager.h

```diff
@@ -38,88 +38,115 @@ class ET_EXPERIMENTAL IOManager {
    * @brief Load the IO manager with method metadata for prefill and
    * decode operations.
    *
-   * @param program The program prefill and decode methods are loaded from.
    * @param prefill_method The prefill method to initialize with.
    * @param decode_method The decode method to initialize with.
    */
   ET_NODISCARD virtual runtime::Error load(
-      const executorch::ET_RUNTIME_NAMESPACE::Program& program,
-      executorch::ET_RUNTIME_NAMESPACE::Method& prefill_method,
-      executorch::ET_RUNTIME_NAMESPACE::Method& decode_method) {
-    (void)program;
+      const std::string& prefill_method,
```

Comment:
Check the prefill_method and decode_method methods exists in module. If there's a typo in the method name, it will be hard to catch it.

Reply:
Module will return an error if IOManager tries to work with a Method that doesn't exist, so it's caught at the Module level.